src/MapViewController.swift

```diff
@@ -76,7 +76,8 @@ open class MapViewController: UIViewController, LocationManagerDelegate {
   var currentRouteMarker: TGMapMarkerId?
   open var shouldFollowCurrentLocation = false
   open var findMeButton = UIButton(type: .custom)
-  open var currentAnnotations: [PeliasMapkitAnnotation : TGMapMarkerId] = Dictionary()
+  open var annotationToMarkerId: [PeliasMapkitAnnotation : TGMapMarkerId] = Dictionary()
```

Comment:
If I'm understanding the intent here, basically you want a reverse lookup in the dictionary (kind me the key associated for this value), correct? 

If that's the case, there's at least two different ways we can solve this with a single dictionary. We can write an extension on top of `Dictionary` to add this functionality, OR we can cast to `NSDictionary` which has more functionality.

The problem with both approaches is we can't assume this dictionary is bijective, so we have to return an array of possible results.

The NSDictionary approach looks like
`let keys = (dictionary as NSDictionary).allKeys(for: "myAwesomeValue") as! [String]` 
which you could then return the keys array. 

The pure swift approach looks something like this:
```
extension Dictionary where Value: Equatable {
  func keysForValue(value: Value) -> [Key] {
    return flatMap { (key: Key, val: Value) -> Key? in
      value == val ? key : nil
    }
  } 
}
```

I'm leaning towards pure swift because pure swift, but either would be good and more memory efficient than having two possibly large dictionaries hanging around.